### PR TITLE
Fixing Swagger to ignore abstract classes

### PR DESCRIPTION
**Pull request by chintan-j-patel**
Abstract class should be ignored from services else DynamicClassCreation::createClassByName throws "Fatal error: Cannot instantiate abstract class"

### CODE REVIEW

File: src/Zynga/Framework/ReflectionCache/V1/ReflectionClasses.hh [hunk 1]
```diff
@@ -6,17 +6,16 @@ use \Exception;
 use \ReflectionClass;
 
 class ReflectionClasses {
```
Comment (stevenramirezgames) [inline src/Zynga/Framework/ReflectionCache/V1/ReflectionClasses.hh]: If we made this a non static class and instead used a factory method to return a particular implementation, I think we can get away from having to inject mocking code into these classes. But that's not something for now.

**Comment by stevenramirezgames** [comment]
@chintan-j-patel looks fine to me. Before accepting, can you see why Travis is failing on the automated build check?

**Comment by tigger0jk** [comment]
Looks good to me

**Comment by stevenramirezgames** [comment]
Once Travis is up and running for this PR, we can approve and merge.

**Comment by stevenramirezgames** [comment]
@chintan-j-patel We have merged up Aidan's PR to try to get around this problem. Can you try closing and re-opening this PR again?

**Pull request merged**